bridge_adaptivity/module/tests/test_mock_engine.py

```diff
@@ -0,0 +1,104 @@
+# coding: utf-8
+from ddt import data, ddt, unpack
+from django.test import TestCase
+from mock.mock import patch
+
+from bridge_lti.models import LtiProvider, LtiUser
+from module.models import (
+    Activity, BridgeUser, Collection, CollectionGroup, Engine, GradingPolicy, Sequence, SequenceItem
+)
+
+
+@ddt
+class TestMockEngine(TestCase):
+    fixtures = ['gradingpolicy.json', 'engine.json']
+
+    @patch('module.tasks.sync_collection_engines.apply_async')
+    def setUp(self, mock_apply_async):
+        self.user = BridgeUser.objects.create_user(
+            username='test',
+            password='test',
+            email='test@me.com'
+        )
+        self.consumer = LtiProvider.objects.create(
+            consumer_name='name',
+            consumer_key='key',
+            consumer_secret='secret',
+        )
+        self.lti_user = LtiUser.objects.create(
+            user_id='some_user', course_id='some_course', email=self.user.email,
+            lti_consumer=self.consumer, bridge_user=self.user
+        )
+        self.client.login(username='test', password='test')
```

Comment:
why do we need client authorization? I cannot find where it is used in the tests.